.github/workflows/build.yml

```
@@ -86,6 +86,14 @@ jobs:
       with:
         path: 'flambda_backend'
 
+    - name: Install AFL (for Linux workers)
+      if: matrix.os != 'macos-latest'
```

```suggestion
      if: matrix.os == 'ubuntu-latest'
```

The not-equal test seems a bit brittle if/when we try and test flamdba2 on windows (which should happen eventually).